apps/web/src/components/borrow/table/TransactionOverview.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useContext } from 'react'
+import {
+  ColumnData,
+  ColumnLayout,
+  Icon,
+  StackLayout,
+  Table,
+  Text,
+} from '@bloxifi/ui'
+import { ThemeContext } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+
+import { FormattedNumber } from '../FormattedNumber'
+
+import { WalletBalance } from '@/containers/WalletContainer'
+
+type TableHeader = 'supplyAPY' | 'healthFactor'
+type TransactionData = { name: TableHeader }
+
+interface Props {
+  /**
+   * Selected asset reserve data
+   */
+  reserveData: WalletBalance
+  /**
+   * List of table headers
+   */
+  headers: TableHeader[]
+}
+
+export const TransactionOverview = ({ reserveData, headers }: Props) => {
+  const themeContext = useContext(ThemeContext)
+  const { t } = useTranslation()
+
+  const transactionData: TransactionData[] = []
+  headers.map(name => transactionData.push({ name }))
+
+  const getColumnValue = (name: TableHeader) => {
+    switch (name) {
+      case 'supplyAPY':
+        return (
+          <ColumnLayout align="flex-end" center>
+            <Text as="span" type="body 1" color="oxfordBlue">
+              <FormattedNumber value={reserveData.supplyAPY} percent />
+            </Text>
+          </ColumnLayout>
+        )
+      case 'healthFactor':
+        return (
+          <StackLayout>
+            <ColumnLayout align="flex-end" center>
+              <Icon name="union" size={16} color={themeContext.buttonDark} />
+              <Icon
+                name="arrow-right"
+                size={15}
+                color={themeContext.buttonDark}
```

Comment:
yeah, we need to handle `Icon.color` similar to (at-least) like `Text.color`

No need, to have themeContext to be used outside of the `packages/ui`.

It will be hard to maintain this, when we are changing colors, since themeContext is type `any`